docs/tests.md

```diff
@@ -0,0 +1,10 @@
+# Tests
+
+## Run behat test 
+```
+make reset-db
+docker-compose exec php sh
```

Comment:
Could be run inside the container as command instead of connect through the shell inside it.